configs/sites/hera/packages.yaml

```diff
@@ -22,11 +22,14 @@ packages:
       prefix: /apps/oneapi
   openmpi:
     externals:
-    - spec: openmpi@4.1.5~cuda~cxx~cxx_exceptions~java+lustre~memchecker+pmi~static~wrapper-rpath schedulers=slurm
-      prefix: /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/openmpi-4.1.5
+    - spec: openmpi@4.1.6~atomics~cuda~cxx~cxx_exceptions~gpfs~internal-hwloc~internal-libevent~internal-pmix~java+legacylaunchers~lustre~memchecker~openshmem~orterunprefix+pmi+romio+rsh~singularity+static+vt+wrapper-rpath fabrics=ucx schedulers=slurm
       modules:
       - gnu/9.2.0
-      - openmpi/4.1.5
+      - openmpi/4.1.6_gnu9.2.0
```

Comment:
Maybe we should try making this the default for all builds (especially Intel) to avoid the issues we've seen with intel@2021.5.0? Not sure. Worth discussing.